src/System.Private.CoreLib/shared/System/IO/FileStream.WinRT.cs

```diff
@@ -57,10 +57,14 @@ private unsafe SafeFileHandle OpenHandle(FileMode mode, FileShare share, FileOpt
             // has been a write on the other end.  We'll just have to deal with it,
             // For the read end of a pipe, you can mess up and 
             // accidentally read synchronously from an async pipe.
-            if (_canRead)
```

Comment:
Not using anymore

Reply:
Yes, the equivalent is `((access & FileAccess.Read) != 0)`.  `_canRead` is an artifact of when I originally wrote this 2 years ago in CoreFX.